src/main/java/org/opensrp/mappers/ExportEventDataMapper.java

```diff
@@ -0,0 +1,95 @@
+package org.opensrp.mappers;
```

Comment:
This package is reserved for mybatis mappers, change package to service

Reply:
Sure